### PR TITLE
Update community-membership.md

### PR DESCRIPTION
**Pull request by sptramer**
Add some requirements to become a _docs_ contributor or technical reviewer, which does not include lines of code.

Signed-off-by: Stephen Tramer <169061+sptramer@users.noreply.github.com>

### CODE REVIEW

**Comment by sptramer** [comment]
**In review until October 8, 12PM PST**.

File: community-membership.md [hunk 1]
```diff
@@ -8,37 +8,60 @@ To contribute, please review our [Code of Conduct](https://github.com/o3de/o3de/
...
-Requirements to become a reviewer for a given Special Interest Group
+Requirements to become a _code_ reviewer for a Special Interest Group (SIG):
+
 * 6+ contributions successfully submitted to O3DE
 * 100+ lines of code changed across all contributions submitted to O3DE
 * 2+ O3DE Reviewers or Maintainers that support promotion from Contributor to Reviewer
 
-You must be nominated (self-nomination is allowed) for promotion from Contributor to Reviewer.
-To do this, an issue should be created on the SIG's repo for the supportive Reviewers and Maintainers to comment and add their support. For example, to nominate a reviewer on behalf of sig-content, an Issue should be created in the [sig-content repository](https://github.com/o3de/sig-content/issues/new/choose).
+Requierments to become a _docs_ reviewer for a SIG:
```
Comment (brakhane) [inline community-membership.md:17]: Typo. Requirements

File: community-membership.md [hunk 2]
```diff
@@ -8,37 +8,60 @@ To contribute, please review our [Code of Conduct](https://github.com/o3de/o3de/
...
+
+* 6+ documentation contributions submitted to o3de.org or O3DE
+* 2+ D&C SIG Reviewers or Maintainers that support promotion, _or_ 1 D&C SIG Reviewer or Maintainer and 1 Maintainer from another SIG.
+
+Docs does _not_ have a "lines of code" or "size of contribution" requirement. Promotions which are rejected will include a concrete requirement for additional contributions that would give the experience required to promote to reviewer.
```
Comment (spotz) [inline community-membership.md:22]: I would say a path to follow to promote to reviewer vs a concrete requirement for additional contributions. For example, working with someone to improve the content of their reviews might be what someone needs to improve on but might be hard to call it a concrete requirement.

File: community-membership.md [hunk 3]
```diff
@@ -8,37 +8,60 @@ To contribute, please review our [Code of Conduct](https://github.com/o3de/o3de/
...
+
+The specialized D&C SIG role of _Documentation Technical Reviewer_ requires:
+
+* Being a Reviewer or Maintainer for a SIG governing the appropriate tech
+* The nomination of 2+ of the appropriate SIG's Reviewers or Maintainers.
+
+You must be nominated (self-nomination is allowed) for promotion from Contributor to Reviewer. To do this, an issue should be created on the SIG's repo for the supportive Reviewers and Maintainers to comment and add their support. For example, to nominate a reviewer on behalf of sig-content, an Issue should be created in the [sig-content repository](https://github.com/o3de/sig-content/issues/new/choose).
 
 Once you have the requisite support, the nomination Pull Request needs to be reviewed and approved during a SIG meeting.  The nominee should plan to attend and add it to the agenda for the SIG to review and, if appropriate, approve.
```
Comment (FiniteStateGit) [inline community-membership.md:31]:
```
The nominee should plan to attend...
```
I think this requirement should be dropped for both roles.  Any given meeting time is going to be inconvenient for some part of the globe and a nominally 'good' time can still fall on the wrong side of a person's working hours.  If there are questions to be addressed, perhaps they could be moved to the original nomination.

File: community-membership.md [hunk 4]
```diff
@@ -8,37 +8,60 @@ To contribute, please review our [Code of Conduct](https://github.com/o3de/o3de/
...
+
 * 6+ contributions successfully submitted to O3DE
 * 100+ lines of code changed across all contributions submitted to O3DE
 * 2+ O3DE Reviewers or Maintainers that support promotion from Contributor to Reviewer
 
-You must be nominated (self-nomination is allowed) for promotion from Contributor to Reviewer.
-To do this, an issue should be created on the SIG's repo for the supportive Reviewers and Maintainers to comment and add their support. For example, to nominate a reviewer on behalf of sig-content, an Issue should be created in the [sig-content repository](https://github.com/o3de/sig-content/issues/new/choose).
+Requierments to become a _docs_ reviewer for a SIG:
+
+* 6+ documentation contributions submitted to o3de.org or O3DE
```
Comment (sptramer) [inline community-membership.md:19]:
```suggestion
* 6+ documentation contributions merged to o3de.org or O3DE

```

File: community-membership.md (hunk 1 above)
Comment (spotz) [reply community-membership.md:17]: Are we having docs reviewer in the SiGs themselves or are we having specialists within the Docs SiG?

File: community-membership.md (hunk 4 above)
Comment (spotz) [reply community-membership.md:19]: I like this change

File: community-membership.md [hunk 5]
```diff
@@ -8,37 +8,60 @@ To contribute, please review our [Code of Conduct](https://github.com/o3de/o3de/
...
 Once you have the requisite support, the nomination Pull Request needs to be reviewed and approved during a SIG meeting.  The nominee should plan to attend and add it to the agenda for the SIG to review and, if appropriate, approve.
 Nomination Pull Requests that do not have the necessary support will be closed after 4 weeks of inactivity.
 
-Requirements to continue to be a Reviewer
-* 4+ Pull Requestss reviewed per month
+Requirements to continue to be a Reviewer:
+
+* 4+ Pull Requests reviewed per month
+
+Requirements to continue to be a Documentation Techncial Reviewer:
```
Comment (spotz) [inline community-membership.md:38]: Are these additional requirements?

File: community-membership.md [hunk 6]
```diff
@@ -8,37 +8,60 @@ To contribute, please review our [Code of Conduct](https://github.com/o3de/o3de/
...
+
+* You serve in the regular tech reviewer rotation and respond to requests in a timely fashion.
 
-If a Reviewer fails to meet the requirements, they will be given a 1 month grace period to get back to the required number of Pull Requests per month.  After the 2nd month, if they haven't returned to the required number of reviews, they will be returned to Contributor status.
+If a Reviewer fails to meet the requirements, they will be given a 1 month grace period.  After the 2nd month, if they haven't returned to the required number of reviews, they will be returned to Contributor status.
 
 ## How to become a maintainer
 
-Requirements to become a maintainer for a given Special Interest Group
+Requirements to become a maintainer for a  Special Interest Group:
```
Comment (Mr-DG-Wick) [inline community-membership.md:46]:
```suggestion
Requirements to become a maintainer for a  Special Interest Group :

```

File: community-membership.md [hunk 7]
```diff
@@ -8,37 +8,60 @@ To contribute, please review our [Code of Conduct](https://github.com/o3de/o3de/
...
-Nomination Pull Requests that do not have the necessary support will be closed after 4 weeks of inactivity.
+* Has been a Reviewer for 2+ months
+* 8+ reviewed Pull Requests in the previous 2 months
+* 2+ SIG Maintainers that support the promotion from Reviewer to Maintainer
+
+You must be nominated (self-nomination is allowed) for promotion from Reviewer to Maintainer. To do this, an issue should be created on the community repo for the supportive Maintainers to comment and add their support. For example, to nominate a maintainer on behalf of sig-content, an Issue should be created in the [sig-content repository](https://github.com/o3de/sig-content/issues/new/choose).
+
+Once you have the requisite support, the nomination Pull Request needs to be reviewed and approved during a SIG meeting.  The nominee should plan to attend and add it to the agenda for the SIG to review and, if appropriate, approve. Nomination Pull Requests that do not have the necessary support will be closed after 4 weeks of inactivity.
```
Comment (Mr-DG-Wick) [inline community-membership.md:61]:
```suggestion
Once you have the requisite support, the nomination pull request needs to be reviewed and approved during a SIG meeting.  The nominee should plan to attend and add it to the agenda for the SIG to review and, if appropriate, approve. Nomination pull requests that do not have the necessary support will be closed after 4 weeks of inactivity.

```

Optional [ I don't know if it was intentional, but capitalization in the middle of a sentence in general is avoided ]

File: community-membership.md [hunk 8]
```diff
@@ -8,37 +8,60 @@ To contribute, please review our [Code of Conduct](https://github.com/o3de/o3de/
...
+
+Requirements to continue to be a Maintainer:
```
Comment (Mr-DG-Wick) [inline community-membership.md:63]:
```suggestion
Requirements to continue to be a Maintainer :

```

Comment (Mr-DG-Wick) [reply community-membership.md:63]: This change is more of a style preference. You may or may not change, but in general having a bit of space makes the doc more organized.

File: community-membership.md (hunk 1 above)
Comment (Mr-DG-Wick) [inline community-membership.md:17]:
```suggestion
Requierments to become a _docs_ reviewer for a SIG :

```
Optional [style preference]

File: community-membership.md [hunk 9]
```diff
@@ -8,37 +8,60 @@ To contribute, please review our [Code of Conduct](https://github.com/o3de/o3de/
 
 ## How to become a reviewer
 
-Requirements to become a reviewer for a given Special Interest Group
+Requirements to become a _code_ reviewer for a Special Interest Group (SIG):
```
Comment (Mr-DG-Wick) [inline community-membership.md:11]:
```suggestion
Requirements to become a _code_ reviewer for a Special Interest Group (SIG) :

```
Optional [style preference]

**Pull request closed without merging**